scripts/hotrod-integration-test.sh

```diff
@@ -28,6 +27,39 @@ if [[ $body != *"Rides On Demand"* ]]; then
   echo "String \"Rides On Demand\" is not present on the index page"
   exit 1
 fi
-docker rm -f "$CID"
```

Comment:
The test needs to shut down containers once it's done, you are leaving them running.